web/views.py

```
@@ -6469,6 +6476,202 @@
     )
 
 
+@login_required
+def membership_checkout(request, plan_id):
```

Ruff (0.8.2)

Missing return type annotation for public function `membership_checkout`

(ANN201)